wrds/sql.py

```diff
@@ -128,6 +124,22 @@ def connect(self):
                 self._password = None
                 raise e
 
+            # Connection successful. Offer to create a .pgpass for the user.
```

Comment:
Perhaps at this point if `self._password` is empty skip the pgpass prompting?
At this point in the code a successful connection and no password can be a sign that pgpass file was used with a non default username

Reply:
Valid point, but I'd rather not infer that the file exists. I can add a check for that before prompting to create one.

Reply:
Actually, I think we're covered. Consider the following (real-life, as in it just happened to me) case: I had a `.pgpass` but then wanted to authenticate with a test username. The only way to do that currently is to specify it when building the connection: 

`db = wrds.Connection(wrds_username='vjeftest')`

I already had a `.pgpass` entry for this user. That trips line 105:

`self.connection = self.engine.connect()`,

connects successfully, and bypasses any prompts to set up a `.pgpass` file.

Alternately, if I do not have an entry for the user in `.pgpass`, I get prompted for a password and then to create the `.pgpass` file, which still succeeds. There are some overwrite/update issues that I believe are being handled in another PR, which are probably outside the scope of changes proposed here.